src/azul/project/hca/__init__.py

```diff
@@ -20,6 +20,13 @@
                                   if config.dss_endpoint != "https://dss.data.humancellatlas.org/v1"
                                   else "files.biomaterial_json")
                     }
+                },
```

Comment:
This needs to be conditional. It should only be active if we're hitting the integration instance of DSS. See above for example.